From nobody Mon Sep 17 00:00:00 2001
From: Alexandre Duret-Lutz <adl@gnu.org>
Date: Mon Dec 26 23:04:24 2005 -0800
Subject: [PATCH] USB: usb-storage support for SONY DSC-T5 still camera
Patch-mainline: 2.6.16-rc3

I've been offered a nice Sony DSC-T5 digital camera, with a USB connection.
Unfortunately it is not recognized by Linux 2.6.14.4's usb-storage.

With the following change I'm able to mount and read my pictures:

Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---

 drivers/usb/storage/unusual_devs.h |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

9f74f19548f3953682b294a080330ea14f46c030
diff --git a/drivers/usb/storage/unusual_devs.h b/drivers/usb/storage/unusual_devs.h
index 5e2afd4..ee958f9 100644
--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -429,11 +429,11 @@ UNUSUAL_DEV(  0x054c, 0x0010, 0x0106, 0x
 		US_FL_SINGLE_LUN | US_FL_NOT_LOCKABLE | US_FL_NO_WP_DETECT ),
 
 /* This entry is needed because the device reports Sub=ff */
-UNUSUAL_DEV(  0x054c, 0x0010, 0x0500, 0x0500, 
-               "Sony",
-               "DSC-T1", 
-               US_SC_8070, US_PR_DEVICE, NULL,
-               US_FL_SINGLE_LUN ),
+UNUSUAL_DEV(  0x054c, 0x0010, 0x0500, 0x0600,
+		"Sony",
+		"DSC-T1/T5",
+		US_SC_8070, US_PR_DEVICE, NULL,
+		US_FL_SINGLE_LUN ),
 
 
 /* Reported by wim@geeks.nl */
-- 
1.1.4.gdcc6

